docs/guide/migrating-from-vuepress.md

```diff
@@ -0,0 +1,105 @@
+# Migrating from VuePress
+
+VitePress follows VuePress config and default theme API where possible to ease the migration path. There are although some features that are not present in vitepress because of differences in the [design goals](../index.md) of the two projects. VitePress aims to have bare minimal features for authoring docs and most features are pushed to the Themes, where else VuePress has those features enabled by plugins.
```

Comment:
> There are although some features that are not present in vitepress because of differences...

Seems to be a typo here, I would go with something like:
> However, there are some features that are not present in VitePress because of...

Reply:
> where else VuePress

guessing you meant:

> whereas VuePress

Reply:
Done, thanks a lot!